configure.ac

```diff
@@ -151,6 +149,10 @@ AC_CHECK_FUNCS([ \
 	uname \
 ])
 
+# AC_FUNC_MALLOC causes problems when cross-compiling.
```

Comment:
AC_FUNC_MALLOC/REALLOC are based on a run-time test which fails when
cross-compiling and we don't use the results from these tests anyway.

I keep them commented because autoscan will recommend we add them.